contracts/sol6/KyberHintHandler.sol

```diff
@@ -165,6 +165,9 @@ abstract contract KyberHintHandler is IKyberHint, Utils5 {
         bytes32[] memory tokenToEthReserveIds,
         uint256[] memory tokenToEthSplits
     ) public view override returns (bytes memory hint) {
+        if (tokenToEthType == TradeType.BestOfAll)
+            checkReserveIdsSplitsIsEmpty(tokenToEthReserveIds, tokenToEthSplits);
```

Comment:
seems like this is checked in verifyData

do we need extra check here.
and in next function?

Reply:
Oh wow somehow we overlooked and I also forgot there's a verifyData function in build hint. Removed the duplicates.